.github/workflows/check-reserved-keywords.yml

```diff
@@ -0,0 +1,27 @@
+name: Check Reserved Keywords
```

Comment:
I believe this file can and should be removed, since `check_keywords` is only implemented for IDAs. I'm putting in a PR for this: https://github.com/openedx/edx-cookiecutters/pull/215